DeAceroDLabs/marketplace-frontend
# fix [DMP-829] Lat and lon without input.

## What is done

[[DMP-829] Lat and lon without input.](https://deacero.atlassian.net/browse/DMP-829)
Lat and lon was the same on all users.
We need to create a task to update the already created users with their correct lat and lon.

## How is done

- Deleted input on Mapfield and use set value when user click on map.
- I notice that the error occurred because onchange on input was not triggered.

## How to test

1. Run the project via `npm start`.
2. Add this line of code on signup.tsx line 37 `  console.log(formState.getValues()["googleMapLatLng"]);` .
3. Create a wrong user, you should see on the console log the correct lat and lon:
![image](https://user-images.githubusercontent.com/81311646/214950912-d911aa6f-f042-4776-b89c-84cc618d2560.png)

Other way to test is to create a new valid user.

## Evidence 

### Before
lat and long always same values
![image](https://user-images.githubusercontent.com/81311646/214951361-2e09c164-2b8e-48a8-b963-e38614c6ce0a.png)
![image](https://user-images.githubusercontent.com/81311646/214951403-d807e108-de68-45de-b147-8ed4a9aaefce.png)

### After
![image](https://user-images.githubusercontent.com/81311646/214951483-c6cdf3f1-0720-45ff-a5e8-edf1e26bd3ea.png)
![image](https://user-images.githubusercontent.com/81311646/214951517-9292558f-bd6f-43f0-8b45-04052cdfb96e.png)
